src/bartholomew.rs

```diff
@@ -114,10 +143,32 @@ pub fn render(req: Request) -> Result<Response> {
                         .clone()
                         .unwrap_or_else(|| DEFAULT_CONTENT_TYPE.to_owned());
 
+                    let disable_page_cache = doc.head.disable_page_cache;
+
                     let data = engine
                         .render_template(doc, config, req.headers().to_owned())
                         .map_err(|e| anyhow!("Rendering {:?}: {}", &content_path, e))?;
 
+                    if !disable_cache {
+                        match disable_page_cache {
+                            Some(true) => {}
+                            _ => {
+                                let index_cache =
```

Comment:
Is it practical to move this out to a named function e.g. `cache_rendered_page`?  The `render` function is already gargantuan and it might be good to minimise further growth...

Reply:
Good point!